SQL_module/SubjectForm_Task2/db_connection.php

```diff
@@ -0,0 +1,12 @@
+<?php
+function dbConnection(){
```

Comment:
Not sure why and what you did here by adding all those files, but usually a PR should be containing only the files which are necessary for covering the targeted functionalities - in this case they should have been for the CSS task only.